src/prototype_room_my.js

```diff
@@ -284,7 +284,31 @@ Room.prototype.getHarvesterAmount = function() {
   return amount;
 };
 
+Room.prototype.visualizz = function() {
```

Comment:
maybe move it to `visualizer.js` file?

Reply:
i could but may be better to place it out of the check of visualizer enabled as the purpose of the module was to visualize layout not datas